Leanplum-SDK/Classes/Internal/FeatureFlag/LPFeatureFlagManager.m

```diff
@@ -7,6 +7,12 @@
 
 #import "LPFeatureFlagManager.h"
 
+@interface LPFeatureFlagManager()
+
+@property (strong, nonatomic) NSSet *enabledFeatureFlags;
```

Comment:
Would you recommend I put the `enabledCounters` configuration in my manager class as well? Right now I've put it in `LPVarCache`, like some other variables I saw.

Reply:
Yeah, let's keep the counter separate from VarCache for sure